frontend/src/Style/Text.elm

```diff
@@ -0,0 +1,35 @@
+module Style.Text exposing (Size(..), center, xxl)
+
+-- ALIGNMENT
+
+
+type Alignment
```

Comment:
This type is unnecessary, just define return the class name in the below functions.